package.json

```diff
@@ -31,6 +31,7 @@
     "primeicons": "^6.0.1",
     "primeng": "^16.7.2",
     "rxjs": "~7.8.0",
+    "toastr": "^2.1.4",
```

Comment:
Any reason why we are using this?
We have created this foundation using primeNG. So we should be using toast from primeNG too?

https://primeng.org/toast

Reply:
i took the toastr service from pickleball initial setup as suggested by Raj, there we were using toastr, also I don't see any primeng module imported in foundation right now, should I go ahead an add primeng.module.ts and use primeng toastr instead of this?

Reply:
Yes use primeng toastrs only. For Foundation we will rely on PrimeNG only
https://primeng.org/toast

Reply:
removed 'toastr' and used primeng toast module